plugins/restapi/v1/utxo.go

```diff
@@ -174,13 +175,14 @@ func outputsResponse(address iotago.Address, includeSpent bool, filterType *iota
 			return nil, errors.WithMessagef(restapi.ErrInternalError, "reading spent outputs failed: %s, error: %s", address, err)
 		}
 
-		for _, spent := range spents {
-			outputIDs = append(outputIDs, spent.OutputID().ToHex())
+		outputIDs = append(outputIDs, make([]string, len(spents))...)
+		for i, spent := range spents {
+			outputIDs[unspentOutputsLength+i] = spent.OutputID().ToHex()
```

Comment:
why not create a new slice first, fill it up with the spents and THEN append it?

Reply:
we can also do that, I took this one from the golang page <https://github.com/golang/go/wiki/SliceTricks#expand>
Should I change it?

Reply:
I would change it. It is the same if you expand with an "empty" slice or a "filled" slice. And the `outputIDs[unspentOutputsLength+i]` makes it error prone I think.